runtime/src/execute_cost_table.rs

```diff
@@ -44,24 +44,24 @@ impl ExecuteCostTable {
         self.table.len()
     }
 
-    /// default program cost, set to ComputeBudget::DEFAULT_UNITS
-    pub fn get_default_units(&self) -> u64 {
-        DEFAULT_UNITS as u64
+    /// default program cost, set to ComputeBudget::DEFAULT_COMPUTE_UNIT_LIMIT
+    pub fn get_default_compute_unit_limit(&self) -> u64 {
+        DEFAULT_INSTRUCTION_COMPUTE_UNIT_LIMIT as u64
     }
 
     /// average cost of all recorded programs
-    pub fn get_global_average_units(&self) -> u64 {
+    pub fn get_global_average_program_cost(&self) -> u64 {
```

Comment:
just a side note, early it was changed from `cost` to `units` for clarity, terms are more consistent since then, `program_cost` might be more intuitive now. Thanks for making change.